.circleci/config.yml.d/amb_util.yml

```diff
@@ -132,3 +132,14 @@ commands:
           path: /tmp/test-logs
           destination: test-logs
 
+  "amb-skip-if-bad-chart-version":
+    steps:
+      - run:
+          name: "Diff changes"
+          command: |
+            thisversion=$(grep version: charts/ambassador/Chart.yaml | awk ' { print $2 }')
+
+            if [[ "<< pipeline.git.tag >>" != "chart-test-v${thisversion}" ]]; then
```

Comment:
i assume `-test` needs to be remove here?

Reply:
Augh. Yes. I'll fix that.